source/includes/documentation/_vue_js_elements.md

```diff
@@ -2301,33 +2301,57 @@ For examples of usage see: [JDI vuetify page tests for snackbars](https://github
 
 ### 5.19 Sparklines
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/sparklines/)
+[Vuetify documentation page](https://v2.vuetifyjs.com/en/components/sparklines/)
 
 - __Java__: _com.epam.jdi.light.vuetify.elements.common.Sparkline.java_
 
 ```java
+    //@FindBy(css = "#HeartRateSparkline svg")
+    @UI("#HeartRateSparkline svg")
+    public static Sparkline heartRateSparkline;
+
     @Test
     public void heartRateSparklineTests() {
```

Comment:
please, update as we have all tests with descriptions :)

Reply:
updated